clusterloader/config/rctest.yaml

```diff
@@ -6,22 +6,22 @@ ClusterLoader:
       tuning: default
       RCs:
         - num: 5
-          image: k8s.gcr.io/pause-amd64:3.0
-          basename: pausepods
+          image: tomcat
```

Comment:
Please don't change that.

I'm fine with adding a separate configs with what you want, but I don't want to run tomact by default.

Reply:
Indeed, you need specific tests to play back real world mix of pods and image types.